README.md

```diff
@@ -35,15 +36,18 @@ The quickest way to get OmiseGO eWallet Server running on macOS and Linux is to
     docker-compose up -d
     ```
 
-For other platforms or for a more advanced setup, see also manual installation below.
+Had a problem during the installation? See [Setup Troubleshooting Guide](docs/setup/troubleshooting.md).
```

Comment:
Encountered a problem during the installation? See the Setup Troubleshoot Guide...